<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ylheq" ignored="0" name="test1" tests="5" failures="0" timestamp="19 十二月 2019 05:57:51 GMT" time="0.067" errors="0">
  <testcase name="test1" time="0.014" classname="com.test.testng.MutliThreadOnXml"/>
  <testcase name="runTimeExceptionSuccess1" time="0.019" classname="com.test.testng.ExceptionTest"/>
  <testcase name="test2" time="0.005" classname="com.test.testng.MutliThreadOnXml"/>
  <testcase name="test3" time="0.003" classname="com.test.testng.MutliThreadOnXml"/>
  <testcase name="runTimeExceptionSuccess2" time="0.002" classname="com.test.testng.ExceptionTest"/>
</testsuite> <!-- test1 -->
